src/routes/results.js

```diff
@@ -1,4 +1,4 @@
-const findAllStartups = require('../find_all_startups.js');
+const findAllStartups = require('../queries/find_all_startups.js');
```

Comment:
this should be changed to 'actions' when we move all functionality out of the routes